modules/apps/change-tracking/change-tracking-lang/src/main/resources/content/Language.properties

```diff
@@ -21,6 +21,7 @@ javax.portlet.title.com_liferay_change_tracking_change_lists_web_portlet_ChangeL
 javax.portlet.title.com_liferay_change_tracking_my_change_lists_web_portlet_MyChangeListsPortlet=My Change Lists
 javax.portlet.title.com_liferay_change_tracking_site_change_lists_history_web_portlet_SiteChangeListsHistoryPortlet=History
 javax.portlet.title.com_liferay_change_tracking_site_change_lists_web_portlet_SiteChangeListsPortlet=Overview
+javax.portlet.title.com_liferay_change_tracking_web_portlet_ChangeListsPortlet=Overview
```

Comment:
@dantewang @ChiZelin 
We only have one ChangeListsPortlet.java in portal, but after your change, we have two entries in the language.properties:
javax.portlet.title.com_liferay_change_tracking_change_lists_web_portlet_ChangeListsPortlet=Change Lists
javax.portlet.title.com_liferay_change_tracking_web_portlet_ChangeListsPortlet=Overview

We should not have two entries, please try to figure out why you need to an a new entry, I guess it may be caused by we renamed this class, but only part of related logic got update.

Tina.